packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -52,8 +53,11 @@ export class LoginPage extends BaseContainer {
 		await this.page.fill( selectors.username, username );
 		await this.page.keyboard.press( 'Enter' );
 		await this.page.fill( selectors.password, password );
-
-		// Enter submits the form and initiates the log in process.
-		await this.page.keyboard.press( 'Enter' );
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( selectors.loginButton ),
+		] );
+		// make sure after logging in that everything stablizes, so we can continue with the next action!
+		await this.page.waitForLoadState( 'load' );
```

Comment:
I think I also changed something in the LoginFlow in #54848:
```
await Promise.all( [ this.page.waitForNavigation(), this.baseflow() ] );
```
Perhaps having both isn't necessary, so I'd leave it up to you as to which one you'd want to keep.

Reply:
Sounds good. I don't think the additional layer of `waitForNavigation()` will trip anything up, but maybe it should be removed from the `LoginFlow` side of things in a separate PR.